t/0-archive.t

```diff
@@ -0,0 +1,34 @@
+# Unit test for TEST_NGINX_ARCHIVE_PATH
```

Comment:
Well, we should avoid requiring a real nginx for running this Perl module's test suite.